### ISSUE TITLE
Release Twisted 16.4.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8743]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8743|
|Type|enhancement|
|Created|2016-08-11 03:01:45Z|
|Branch|https://github.com/twisted/twisted/tree/release-16.4.0-8743|

<details><summary>Searchable metadata</summary>

```
trac-id__8743 8743
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Twisted_16_4 Twisted 16.4
branch__release_16_4_0_8743 release-16.4.0-8743
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470884505408994 1470884505408994
changetime__1472349878593274 1472349878593274
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Please review.

The failing asyncio builders are because that landed in trunk; not 16.4.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc set owner to @hawkowl|
|-|-|

This looks OK.

There are quite a lot of commits done in this release since the last release.
Amazing!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 2fec41db6f1a30748283f28897a29d0b6dd1c082
```
#!CommitTicketReference repository="" revision="2fec41db6f1a30748283f28897a29d0b6dd1c082"
Merge branch 'release-16.4.0-8743': Release Twisted 16.4.0

Author: hawkowl
Reviewer: rodrigc
Fixes: #8743
```
